src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -49,25 +62,27 @@ export const RegistrationFormActions = ({ tabNumber, setTabNumber, validateForm
         ['registration', updateRegistrationResponse.data.identifier],
         updateRegistrationResponse.data
       );
-      const newErrors = validateForm(updateRegistrationResponse.data);
-      setTouched(setNestedObjectValues(newErrors, true));
       dispatch(setNotification({ message: t('feedback.success.update_registration'), variant: 'success' }));
+      if (isLastTab) {
+        history.push(getRegistrationLandingPagePath(values.identifier));
+      } else {
+        const newErrors = validateForm(updateRegistrationResponse.data);
+        setTouched(setNestedObjectValues(newErrors, true));
+      }
     }
     setIsSaving(false);
 
     return isSuccess;
   };
 
-  const onClickSaveAndPresent = async () => {
-    const registrationIsUpdated = await saveRegistration(values);
-    if (registrationIsUpdated) {
-      history.push(getRegistrationLandingPagePath(values.identifier));
+  const handleSaveClick = async () => {
+    if (compareRegistrationAndValues(persistedRegistration, values) && isNviCandidate) {
```

Comment:
pirk, men ville snudd om på denne, slik at vi unngår å kalle det som kommer til å bli en tung funksjon om det uansett ikke er en NVI-kandidat
```suggestion
    if (isNviCandidate && compareRegistrationAndValues(persistedRegistration, values)) {
```